http/document_service.go

```diff
@@ -72,30 +80,34 @@ func NewDocumentHandler(b *DocumentBackend) *DocumentHandler {
 }
 
 type documentResponse struct {
-	Links map[string]string `json:"links"`
+	Links         map[string]string `json:"links"`
+	CreatedAt     string            `json:"createAt"`
```

Comment:
These should be `createdAt` and `updatedAt`, to be consistent with dashboards and tasks. These should also be embedded within dashboard in the response.

Reply:
The swagger also needs to be updated for this.